auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -96,18 +101,25 @@ public View onCreateView(LayoutInflater inflater,
 
         View v = inflater.inflate(R.layout.fui_register_email_layout, container, false);
 
+        // Get configuration
+        AuthUI.IdpConfig emailConfig = ProviderUtils.getConfigFromIdps(
+                getFlowParams().providerInfo, EmailAuthProvider.PROVIDER_ID);
+        mRequireName = emailConfig.getParams().getBoolean(ExtraConstants.EXTRA_REQUIRE_NAME, true);
```

Comment:
Can we make this a local variable? Adding a field with an extra default value scares me. 😄